### PR TITLE
Added fake timereport service

### PR DESCRIPTION
**Pull request by Akram-Mansour**
Added get, getsingel, delete and save functions

### CODE REVIEW

**Comment by AhmadS91** [comment]
+2

**Comment by AminSaad** [comment]
+2

File: src/services/fakeTimeReportService.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const timeReports = [
+  {
+    id: 1,
+    date: "2020-03-23",
+    startingTime: "08.00",
+    endTime: "17.00"
```
Comment (AhmadS91) [inline src/services/fakeTimeReportService.js]: Lägg till aktiviteter

**Review by AminSaad — APPROVED** [review]
+2

**Review by ghost — APPROVED** [review]
+2

**Pull request merged**